decai/decai.r2.js

```diff
@@ -256,10 +256,6 @@ You can write your custom decai commands in your ~/.radare2rc file.
         }
 
         try {
-            const o = JSON.parse(res);
```

Comment:
Why

Reply:
The error will just be catched in `catch(e) {` so no need to do this?

Reply:
No. The trycatch captures javascript errors, like invalid json, the other part is related to the huggingface internal errors which are reported in the json response. So this part is necessary